testapi.pm

```diff
@@ -1559,9 +1530,9 @@ Default hold time is 0.15s
 
 =cut
 
-sub mouse_click {
-    my $button = shift || 'left';
-    my $time = shift || 0.15;
+sub mouse_click ($button = undef, $time = undef) {
+    $button //= 'left';
+    $time //= 0.15;
```

Comment:
Strictly, if `$time` was 0, then it would have been set to `0.15` in the original code. Now only if it's undef.

Reply:
ok, I just wanted to mention it.